src/index.tsx

```diff
@@ -23,7 +22,7 @@ const SettingError: React.FC = () => {
   useEffect(() => {
     i18n.changeLanguage(kintone.getLoginUser().language);
   }, [i18n]);
-  return <Alert isVisible text={t('errors.plugin_no_setting_error')} />;
+  return <></>;
```

Comment:
```suggestion
  return <div style={{ color: 'red' }}>{t('errors.plugin_no_setting_error')}</div>;
```
削除より、こういう感じでエラーメッセージを返してあげた方はいかがでしょうか